packages/mainsail/source/helpers/format-units.test.ts

```diff
@@ -0,0 +1,16 @@
+import { formatUnits } from "./format-units";
+
+describe("formatUnits", async ({ assert, it, loader }) => {
+	it("should format the value to wei", () => {
+		assert.equal(formatUnits("1", "wei"), 1);
+		assert.equal(formatUnits("1000000000", "gwei"), 1);
+		assert.equal(formatUnits("100000000", "ark"), 1);
```

Comment:
should this be 1e18 to 1?

```suggestion
		assert.equal(formatUnits("1000000000000000000", "ark"), 1);
```